### DISCUSSION TITLE
Clarification on NEKRS_CACHE_BCAST + NEKRS_LOCAL_TMP_DIR

### DISCUSSION THREAD

**Discussion by abettini99** [Nek5000/nekRS#574] [Q&A] [answered]
Hi,

I'm new to working on HPC codes. Is the purpose of the NEKRS_CACHE_BCAST + NEKRS_LOCAL_TMP_DIR variables to ensure that all worker nodes have access to the same files found after compiling a test case in a common location across all nodes? NEKRS_CACHE_BCAST being the flag to compiles files not in the .cache folder but to NEKRS_LOCAL_TMP_DIR?

**Comment by stgeke** [comment, marked as answer]
The `NEKRS_CACHE_BCAST` feature is utilized to broadcast the cache over the network, directing it to a node-local disk. This reduces the load on the shared filesystem during the setup phase. It is recommended to use this feature when deploying thousands of compute nodes.
